apps/communications/contacts/js/views/form.js

```diff
@@ -711,11 +711,20 @@ contacts.Form = (function() {
       isIceContact(currentContact, function(result) {
         if (result === true) {
           var msgId = 'ICEContactDelTel';
-          var phoneNumberInput = document.getElementById('number_0');
-          var phoneNumberValue = phoneNumberInput &&
-                                                phoneNumberInput.value.trim();
+          var telInputs = document.querySelectorAll('.textfield[type="tel"]');
+          var hasNumber = false;
+
+          for (var i = 0, len = telInputs.length; i < len; i++) {
+            // The second comparison is needed because the template used is not
+            // a real <template> so the DOM inside it is active. This code
+            // should still work when the template is rewritten as a real one.
+            if (telInputs[i].value && telInputs[i].value != '#value#') {
```

Comment:
you can avoid this check if the selector in line 714 avoids selecting the template, for instance

:not([data-template]) .textfield[type="tel"]